tutorials/printing.adoc

```diff
@@ -1,83 +1,213 @@
-Tisk
-----
+= Tisk
+Marián Hlaváč <marian.hlavac@fit.cvut.cz>
+:toc:
 
-OctoPrint
-~~~~~~~~~
+Tento týden se konečně dostaneme k tomu, na co jsme se všichni jistě celou dobu těšili. 
+Využijeme získané teoretické znalosti a uplatníme je prakticky.
 
-Při kalibračním a dalším tisku budeme používat webový ovládací interface
-http://octoprint.org[OctoPrint].
+Seznámíme se s tiskárnou, vysvětlíme si, k čemu jsou určeny její jednotlivé části a připravíme ji k tisku. 
+Seznámíme se s nástroji, kterými budeme přenášet tisková data a spouštět tisk a závěrečně na tiskárnách něco vytiskneme.
+ 
+_Pokud již 3D tiskárny znáte, můžete některé kapitoly v textu vynechat, ale nevynechávejte kapitolu <<_příprava-tiskárny>>, je povinná pro všechny. Až pak můžete pokračovat čtením xref:_zadání-úkolů[zadaných úkolů]._
```

Comment:
tahle kotva minimálně na githubu úplně nefunguje

Reply:
No, tohle upřímně nevím jak vyřešit. Z nějakého důvodu na courses.fit.cvut.cz fungujou headingy s podtržítkem a na githubu ne. Co na to @jirutka ?

Reply:
Používat generované identifikátory tímto způsobem je dost nepraktické. Buď použij v xref ten nadpis tak, jak je, tj. `<<Příprava tiskárny>>` (aby bylo snadné najít jeho výskyt, když bude chtít někdo daný nadpis změnit), nebo mu přiřaď explicitní ID (`[[priprava-tiskarny]]`), a to použij v xref. Asciidoctor prefixuje generované identifikátory, aby se vyhnul potenciálním kolizím. Na GH mají asi nějakou starou verzi, která to ještě by default nedělala.

> xref:_zadání-úkolů[zadaných úkolů].

Tohle můžeš zapsat jako `<<Zadání úkolů, zadaných ukolů>>`.

Doporučuju používat `<< ... >>` syntaxi pro reference uvnitř dokumentu a `xref` pro reference mezi dokumenty (pro přehlednost).

Reply:
@jirutka ok thx, tohle bychom měli napsat do manuálu course pages taky.